egs/chime4/s5_1ch/local/run_dnn.sh

```diff
@@ -147,7 +147,7 @@ srcdir=exp/tri4a_dnn_tr05_multi_${train}
 acwt=0.1
 
 # First we generate lattices and alignments:
-# gawk must be installed to perform awk -v FS="/" '{ print gensub(".gz","","",$NF)" gunzip -c "$0" |"; }' in
```

Comment:
you can actually remove these comments about gawk, this should no longer be necessary.

Reply:
Do you mean remove the entire comment here or just the words about gawk?

Reply:
about gawk.